var/spack/repos/builtin/packages/openspeedshop/package.py

```diff
@@ -130,84 +138,123 @@ class Openspeedshop(Package):
     depends_on("cbtf", when='+cbtf')
     depends_on("cbtf-krell", when='+cbtf')
     depends_on("cbtf-argonavis", when='+cbtf+cuda')
-    depends_on("mrnet@5.0.1:+lwthreads+krellpatch", when='+cbtf')
+    depends_on("mrnet@5.0.1:+lwthreads", when='+cbtf')
+
+    # We don't need this for building, but to complete the
+    # runtime module file
+    # It is required in cbtf, so it will be available and not
+    # cause another rebuild
+
+    depends_on("xerces-c@3.1.1:", when='+cbtf')
```

Comment:
you probably missed my comment above. If you don't need this for build, then you should adjust dependency type, probably

```
depends_on("xerces-c@3.1.1:", when='+cbtf',  type='run')
```